addon/globalPlugins/webAccess/gui/rule/editor.py

```diff
@@ -921,14 +921,14 @@ def initData(self, context: Mapping[str, Any]) -> None:
 		rule = context.get("rule")
 		data = context.setdefault("data", {}).setdefault(
 			"rule",
-			rule.dump() if rule else OrderedDict()
+			rule.dump() if rule else {}
 		)
 		mgr = context["webModule"].ruleManager if "webModule" in context else None
 		if not rule and mgr and mgr.nodeManager:
 			node = mgr.nodeManager.getCaretNode()
 			while node is not None:
 				if node.role in formModeRoles:
-					data["rule"].setdefault("properties", {})["formMode"] = True
+					context["data"]["rule"].setdefault("properties", {})["formMode"] = True
```

Comment:
@aaclause,
Oops, my bad! Thanks for pointing this out!
Please change to `data.setdefault("properties", {}}["formMode"] = True` for conciseness as `data` is already set to context["data"]["rule"] on line 922.